src/subgraphs/getData.ts

```diff
@@ -19,13 +20,15 @@ interface DataRecoveryState extends BaseState {
   needsSemanticUpdate: boolean;
   semanticTask: string; 
   isPossible: string;
+  status: NodeStatus | null;
```

Comment:
Why do we need NodeStatus in the state?

Reply:
Can't we just create it whenever we are going to do a response?

Reply:
Added it just in case it needed to be accessed in the future. Can remove it, no problem